src/Components/Grants/GrantsIndex.js

```diff
@@ -0,0 +1,203 @@
+  const currentOrganizationId =
+    currentOrganizationStore.currentOrganization?.id;
+  // const { grant_id: grantId } = useParams();
+  const sensors = useSensors(
+    useSensor(PointerSensor)
+    // This breaks forms nested under drag and drop! The space key triggers
+    // this sensor. TODO: Circle back to this!
+    // useSensor(KeyboardSensor, {
+    //   coordinateGetter: sortableKeyboardCoordinates,
+    // })
+  );
+
+  const [searchFilters, setSearchFilters] = useState({
+    title: "",
+  });
+
+  const columns = [
+    { Header: "Deadline", accessor: "deadline" },
+    { Header: "Title", accessor: "title" },
+    { Header: "Funding Org", accessor: "funding_org_name" },
+    { Header: "Purpose", accessor: "purpose" },
+    { Header: "Date Created", accessor: "created_at" },
+    { Header: "Last Modified", accessor: "updated_at" },
+  ];
+
+  const dropDownProps = {
+    labelText: "Further Actions",
+    placeholder: "Pick One",
+    options: [
+      { value: "MARK_AS_SUBMITTED", label: "Mark as Submitted" },
+      { value: "MARK_AS_SUCCESSFUL", label: "Mark as Successful" },
+      { value: "MARK_AS_COPY", label: "Mark as Copy" },
+      { value: "ARCHIVE", label: "Archive" },
+    ],
+  };
+
+  useEffect(() => {
+    if (organizationClient)
+      getAllGrants(organizationClient)
+        .then((grants) => {
+          setGrants(grants);
+          setLoading(false);
+        })
+        .catch((error) => console.log(error));
+  }, [organizationClient]);
+
+  const filteredGrants = useMemo(() => {
+    return grants.filter((grant) => {
+      const matchesTitle = grant.title
+        .toLowerCase()
+        .includes(searchFilters.title.toLowerCase());
+      return matchesTitle;
+    });
+  }, [grants, searchFilters]);
+
+  useEffect(() => {
+    if (organizationClient) {
+      setFilteredGrantsByTabName(filteredGrants);
+    }
+  }, [organizationClient, filteredGrants]);
+
+  const filterGrantsByTabName = (tabSelect) => {
+    const filteredByTab = filteredGrants.filter((grant) => {
+      if (tabSelect === "All") {
+        return grant;
+      } else if (tabSelect === "Archived") {
+        return grant.archived === true;
+      } else if (tabSelect === "Drafts") {
+        return grant.submitted === false;
+      } else if (tabSelect === "Successful") {
+        return grant.successful === true;
+      } else if (tabSelect === "Submitted") {
+        return grant.submitted === true;
+      }
+      return grant;
+    });
+    setFilteredGrantsByTabName(filteredByTab);
+  };
+
+  if (errors.length) {
+    console.error(errors);
+    return <p>Error! {errors.map((error) => error.message)}</p>;
+  } else if (loading) {
+    return <h1>Loading....</h1>;
+  }
+
+  return (
+    <div className="GrantsIndex">
+      <section className="GrantsIndex__Overview">
+        <header className="GrantsIndex__Header">
+          <h1 className="GrantsIndex__HeaderText">All Grants</h1>
+        </header>
```

Comment:
Since we don't have any other styling requirements other than needing the h1, I'd recommend removing the header altogether:

```suggestion
        <h1 className="GrantsIndex__HeaderText">All Grants</h1>
```